stl/src/syserror.cpp

```diff
@@ -38,6 +38,7 @@ namespace {
         {ERROR_INVALID_DRIVE, errc::no_such_device},
         {ERROR_CURRENT_DIRECTORY, errc::permission_denied},
         {ERROR_NOT_SAME_DEVICE, errc::cross_device_link},
+        {ERROR_NO_MORE_FILES, errc::no_such_file_or_directory},

```

Comment:
Is this not `errc::too_many_files_open`?

Reply:
`ERROR_NO_MORE_FILES` is returned by a file iterator, for example, Windows returns this error when there are no more files in the call to `FindNextFileA` https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-findnextfilea

Reply:
I could not get this error after more than 100'000 files open, so maybe not.

Reply:
I don't like this then; it's _not_ an error, it's certainly not `no_such_file_or_directory`